data/data_partners.json

```diff
@@ -0,0 +1,12 @@
+{
+  "NV": {
+    "nv-nevada-clean-energy-fund": {
```

Comment:
I considered nesting this under `state` in case we want to add different levels of partners (state, utility, etc), but ultimately that seemed overkill, and I think it's fine to give these partners credit state-wide even if they only review incentives for a specific utility